File: mohit/python/chap 04 pattern/rough.py
n=int(input("enter the number: "))
for i in range(0,n+1):
    for j in range(0,(n-1)*2):
        if(j>=(n+1)-i and j<=3+i):
            print(" ",end=" ")
        else:
            print("*",end=" ")
        
    print()      

File: mohit/C Programs/rough.c
#include<stdio.h>
main(){
    int r,c,a[r][c],i,j;
    printf("enter the row number: ");
    scanf("%d",&r);
    printf("enter the column number: ");
    scanf("%d",&c);
    for(i=0;i<r;i++){
        for(j=0;j<c;j++){
            scanf("%d",&a[i][j]);
        }
    }
    printf("\n");
    for(i=0;i<r;i++){
        for(j=0;j<c;j++){
            printf("%d",a[i][j]);
        }
    }
    printf("\n");
}